pyproject.toml

```diff
@@ -61,7 +61,9 @@ django-split-settings = "^1.2.0"
 pexpect = "^4.9.0"
 ansible-runner = ">=2.3"
 python-gnupg = "^0.5.2"
-autobahn = { git = "https://github.com/crossbario/autobahn-python.git", rev = "v24.4.2" }
+# indirectly needed by twisted, this version avoids issues with newer versions of setuptools:
+# https://github.com/crossbario/autobahn-python/pull/1632
+autobahn = ">=24.4.2"
```

Comment:
I have noticed the version constraint is changed, better to pin the version without allowing unintentional upgrades. @ptoscano